lib/inputs/binaries/static/index.ts

```diff
@@ -1 +1,21 @@
-// TODO
+import { ExtractAction, ExtractedLayers } from "../../../extractor/types";
+import { streamToHash } from "../../../stream-utils";
+
+export const getNodeBinariesFileContentAction: ExtractAction = {
+  actionName: "node",
+  fileNamePattern: "**/*/node",
+  callback: streamToHash,
+};
+
+export function getBinariesHashes(extractedLayers: ExtractedLayers): string[] {
+  const hashes: string[] = [];
+  for (const fileName of Object.keys(extractedLayers)) {
+    for (const actionName of Object.keys(extractedLayers[fileName])) {
+      if (actionName !== getNodeBinariesFileContentAction.actionName) {
+        continue;
+      }
+      hashes[fileName] = extractedLayers[fileName][actionName];
```

Comment:
I assume that's the exact expected format :)

Reply:
what expected format?

Reply:
the format that registry expects in the end?

Reply:
if you're talking about Registry's expectations in the k8s flow - these are not set yet, and still need to be defined. I would rely on the k8s-agent to make the necessary adjustments.
if you're talking about Registry's expectations in the CLI flow - these are not used yet, and I'm more than happy to make adjustments there as we go instead of being fixated on what exists at the moment